conf/README.md

```diff
@@ -210,6 +210,10 @@ IBM Cloud related section under AUTH config.
   [documentation](https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-uhc-hmac-credentials-main).
 * `ibm_cos_secret_access_key` - COS secret key (Follow the same documentation link above!)
 
+DEPLOYMENT section:
+* `create_ibm_cos_secret`: If this value is set to True (by default), the COS
+  secret is created. If False, it will not be created.
+
```

Comment:
Shouldn't this rather be located in `DEPLOYMENT` section (line 60)?

Reply:
Yep, you're right ;) Will move it ;)